physics/cu_gf_driver.F90

```diff
@@ -86,15 +86,15 @@ subroutine cu_gf_driver_run(ntracer,garea,im,km,dt,flag_init,flag_restart,&
      &                    spp_wts_cu_deep
       real(kind=kind_phys) :: spp_wts_cu_deep_tmp
 
-      logical, intent(in) :: do_cap_suppress
+      logical, intent(in) :: do_cap_suppress, do_smoke_transport
       real(kind=kind_phys), parameter :: aodc0=0.14
       real(kind=kind_phys), parameter :: aodreturn=30.
       real(kind=kind_phys) :: dts,fpi,fp
       integer, parameter :: dicycle=0 ! diurnal cycle flag
       integer, parameter :: dicycle_m=0 !- diurnal cycle flag
       integer            :: ishallow_g3 ! depend on imfshalcnv
 !-------------------------------------------------------------
-   integer      :: its,ite, jts,jte, kts,kte
+   integer      :: its,ite, jts,jte, kts,kte, nchem
```

Comment:
`nchem` is missing an intent. By default, variables without declared intents are inout in Fortran, which is a mismatch with the metadata that has it as in. I would suggest moving the nchem declaration to line 98 so that it has the proper intent.

Reply:
@grantfirl The nchem has been declared as intents in. Thanks.